hayeon/baekjoon/스택/10773-제로.js

```diff
@@ -0,0 +1,52 @@
+const readline = require("readline");
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+const input = [];
+
+rl.on("line", function (line) {
+  input.push(line);
+}).on("close", function () {
+  // 풀이
+  const testCase = +input[0];
+  const stack = [];
+
+  for (let i = 1; i <= testCase; i++) {
+    const value = Number(input[i]);
+
+    if (value !== 0) {
+      stack.push(value);
+    } else {
+      stack.pop();
+    }
+  }
+
+  const result = stack.reduce((acc, cur) => acc + cur, 0);
+
+  console.log(result);
```

Comment:
```suggestion
  const numList = input.slice(1).map(Number);
  const stack = [];

  numList.forEach((num) => {
    if (!num) stack.pop();
    else stack.push(num);
  });

  const result = stack.reduce((acc, cur) => acc + cur, 0);

  console.log(result);
```
- 풀이 잘봤습니다 🙇‍♂️
- slice를 통해서 testCase를 잘라줘서 이를 forEach로 활용하면 좀 더 가독성 좋은 코드를 작성할 수 있습니다 🙇‍♂️

Reply:
slice를 통해서 자르는 법이 있었군요!! 저번에 한번 썼었는데 생각도 못하고 있었네요.... 리마인드 감사합니다!!👍🏻